contracts/src/core/EigenDAThresholdRegistry.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {EigenDAThresholdRegistryStorage} from "./EigenDAThresholdRegistryStorage.sol";
+import {IEigenDAThresholdRegistry} from "../interfaces/IEigenDAThresholdRegistry.sol";
+import {OwnableUpgradeable} from "@openzeppelin-upgrades/contracts/access/OwnableUpgradeable.sol";
+import {BitmapUtils} from "eigenlayer-middleware/libraries/BitmapUtils.sol";
+import "../interfaces/IEigenDAStructs.sol";
+
+contract EigenDAThresholdRegistry is EigenDAThresholdRegistryStorage, OwnableUpgradeable {
+
+    constructor() {
+        _disableInitializers();
+    }
+
+    function initialize(
+        address _initialOwner,
+        bytes memory _quorumAdversaryThresholdPercentages,
+        bytes memory _quorumConfirmationThresholdPercentages,
+        bytes memory _quorumNumbersRequired,
+        uint16[] memory _versions,
+        VersionedBlobParams[] memory _versionedBlobParams,
+        SecurityThresholds memory _defaultSecurityThresholdsV2
+    ) external initializer {
+        _transferOwnership(_initialOwner);
+
+        quorumAdversaryThresholdPercentages = _quorumAdversaryThresholdPercentages;
+        quorumConfirmationThresholdPercentages = _quorumConfirmationThresholdPercentages;
+        quorumNumbersRequired = _quorumNumbersRequired;
+        defaultSecurityThresholdsV2 = _defaultSecurityThresholdsV2;
```

Comment:
is there no update function for `defaultSecurityThresholdsV2`?

Reply:
good call